source/player_guide/missions_info/missions/kill_the_traitor.rst

```diff
@@ -9,12 +9,7 @@ Category
 
 .. topic:: Description
 
-   ⚠️ - ⚠️ - ⚠️ - ⚠️ - ⚠️
-
-   🚨 PLACEHOLDER 🚨
-
-   ⚠️ - ⚠️ - ⚠️ - ⚠️ - ⚠️
-
+Note: You can find Intel on the Traitor’s family from dead enemy soldiers. This can be used by Petros to blackmail the Traitor into staying quiet. 
```

Comment:
also has to be indented once if it belongs to the `topic::description`. If not you either have to remove the `.. topic::Description` because topics can not be empty, or ad a placeholder like `WiP` or `soon`